src/widget/wmainmenubar.cpp

```diff
@@ -521,7 +521,10 @@ void WMainMenuBar::initialize() {
     // HELP MENU
     QMenu* pHelpMenu = new QMenu(tr("&Help"), this);
 
-    QString externalLinkSuffix = " =>";
+    QString externalLinkSuffix;
+#ifndef __APPLE__ // external links don't have icons on macOS
```

Comment:
accoring to ....

Reply:
acording to screenshots from @foss- and web search for 'macos help menu'